libs/wire-api/src/Wire/API/User.hs

```diff
@@ -1151,3 +1155,31 @@ instance S.ToSchema ListUsersQuery where
           & S.description ?~ "exactly one of qualified_ids or qualified_handles must be provided."
           & S.properties .~ InsOrdHashMap.fromList [("qualified_ids", uids), ("qualified_handles", handles)]
           & S.example ?~ toJSON (ListUsersByIds [Qualified (Id UUID.nil) (Domain "example.com")])
+
+-----------------------------------------------------------------------------
+-- 2nd Factor Auth
+-- todo(leif): roundtrip tests
+-- todo(leif): golden tests
```

Comment:
yes, but please get the swagger json plus screenshots uploaded to the tech spec first.